pkgs/development/libraries/webrtc-audio-processing/default.nix

```diff
@@ -50,6 +50,6 @@ stdenv.mkDerivation rec {
     license = licenses.bsd3;
     # https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/blob/master/webrtc/rtc_base/system/arch.h
     # x86-32 disabled due to https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/issues/5
-    platforms = intersectLists platforms.unix (platforms.aarch64 ++ platforms.mips ++ platforms.riscv ++ platforms.x86_64);
+    platforms = intersectLists platforms.unix (platforms.armv7 ++ platforms.aarch64 ++ platforms.mips ++ platforms.riscv ++ platforms.x86_64);
```

Comment:
Maybe `subtractLists platforms.i686 platforms.unix` should be used instead?

Reply:
Looks like even i686 is OK: https://gitlab.freedesktop.org/pulseaudio/webrtc-audio-processing/-/blob/master/webrtc/rtc_base/system/arch.h#L31

Reply:
It doesn't build on i686-linux though

Reply:
Sounds like a bug here? Arch Linux 32 does have the package.

Reply:
> Arch Linux 32 does have the package.

afaiu they build for pentium4 and for i686 they only have the older version like we do